### ISSUE TITLE
ToDo: diffs FF122-FF123

### ISSUE THREAD

**Issue by earthlng** [arkenfox/user.js#1814]
FF123 [release date](https://whattrainisitnow.com/calendar/): Feb. 20th 2024

[FF123 release notes](https://www.mozilla.org/firefox/123.0/releasenotes/)
[FF123 for developers](https://developer.mozilla.org/docs/Mozilla/Firefox/Releases/123)
[FF123 security advisories](https://www.mozilla.org/security/advisories/mfsa2024-05/)

---

89 diffs ( 44 new, 30 gone, 15 different )

### new in v123.0:
- [ ] pref("browser.link.force_default_user_context_id_for_external_opens", false); [1874599](https://bugzilla.mozilla.org/show_bug.cgi?id=1874599)
- [x] [1853996](https://bugzilla.mozilla.org/show_bug.cgi?id=1853996) [meta] Revamp Clear Data Dialog
   - for FF128 migration: https://github.com/arkenfox/user.js/pull/1816/commits/e7fa19f43c4f87681cf98fadeaf09707bc16e1aa
   - pref("privacy.clearOnShutdown_v2.cache", true);
   - pref("privacy.clearOnShutdown_v2.cookiesAndStorage", true);
   - pref("privacy.clearOnShutdown_v2.historyFormDataAndDownloads", true);
   - pref("privacy.clearOnShutdown_v2.siteSettings", false);
- FYI: pref("privacy.sanitize.useOldClearHistoryDialog", true);

### removed, renamed or hidden in  v123.0:
- [x] pref("browser.ping-centre.telemetry", true); [1868988](https://bugzilla.mozilla.org/1868988) - https://github.com/arkenfox/user.js/pull/1816/commits/e037014a56ed22a7de39ebef6efe9dfe0c325549

### changed in  v123.0:
- `7016` pref("privacy.partition.network_state.ocsp_cache", true); // prev: false [1874143](https://bugzilla.mozilla.org/show_bug.cgi?id=1874143) - notated in v122 https://github.com/arkenfox/user.js/pull/1764/commits/5916d0cb4e78bad33847d2648a8e65632fcfc9c1

---

### ignore

<details><summary>click me for details</summary><p>

==NEW
```js
pref("apz.minimap.enabled", false);
pref("browser.shopping.experience2023.autoOpen.enabled", false);
pref("browser.shopping.experience2023.autoOpen.userEnabled", true);
pref("browser.shopping.experience2023.showKeepSidebarClosedMessage", true);
pref("browser.shopping.experience2023.sidebarClosedCount", 0);
pref("browser.tabs.cardPreview.delayMs", 1000);
pref("browser.tabs.cardPreview.enabled", false);
pref("browser.tabs.cardPreview.showThumbnails", true);
pref("browser.urlbar.suggest.yelp", true);
pref("cookiebanners.ui.desktop.showCallout", false);
pref("devtools.debugger.javascript-tracing-values", false);
pref("devtools.inspector.rule-view.focusNextOnEnter", true);
pref("extensions.formautofill.addresses.capture.requiredFields", "street-address,postal-code,address-level1,address-level2");
pref("extensions.formautofill.heuristics.captureOnPageNavigation", false);
pref("identity.fxaccounts.oauth.enabled", false);
pref("javascript.options.destructuring_fuse", true);
pref("javascript.options.wasm_moz_intgemm", false);
pref("layout.css.grid-subgrid-baselines.enabled", false);
pref("layout.cursor.disable-for-popups", true);
pref("media.eme.wmf.clearkey.enabled", false);
pref("media.eme.wmf.use-mock-cdm-for-external-cdms", false);
pref("network.cookie.prevent_set_cookie_from_multipart", true);
pref("network.dns.native_https_query", false);
pref("network.dns.native_https_query_win10", false);
pref("network.dns.native_https_timeout_android", 20000);
pref("network.fetch.cache_partition_cross_origin", true);
pref("network.http.useragent.freezeCpu", false);
pref("network.protocol-handler.external.firefox", false);
pref("network.protocol-handler.external.firefox-private", false);
pref("network.standard_content_type_parsing.response_headers", true);
pref("screenshots.browser.component.last-saved-method", "download");
pref("security.tls.enable_kyber", false);
pref("services.sync.prefs.sync.privacy.clearOnShutdown_v2.cache", true);
pref("services.sync.prefs.sync.privacy.clearOnShutdown_v2.cookiesAndStorage", true);
pref("services.sync.prefs.sync.privacy.clearOnShutdown_v2.downloads", true);
pref("services.sync.prefs.sync.privacy.clearOnShutdown_v2.historyFormDataAndDownloads", true);
pref("services.sync.prefs.sync.privacy.clearOnShutdown_v2.siteSettings", true);
pref("ui.new-webcompat-reporter.reason-dropdown.randomized", true);
```

==REMOVED, RENAMED or HIDDEN
```js
pref("apz.scrollend-event.content.enabled", true);
pref("browser.ping-centre.log", false);
pref("browser.search.searchEngineRemoval", "https://support.mozilla.org/1/firefox/%VERSION%/%OS%/%LOCALE%/search-engine-removal");
pref("devtools.inspector.showRulesViewEnterKeyNotice", true);
pref("dom.importMaps.enabled", true);
pref("dom.mouse_event.region.enabled", false);
pref("dom.webidl.crosscontext_hasinstance.enabled", false);
pref("editor.inline_style.range.compatible_with_the_other_browsers", true);
pref("editor.join_split_direction.force_use_compatible_direction", "");
pref("editor.join_split_direction.force_use_traditional_direction", "");
pref("gfx.canvas.remote.texture-timeout-ms", 10000);
pref("javascript.options.watchtower.megamorphic", true);
pref("layout.css.abs-sign.enabled", true);
pref("layout.css.color-mix.color-spaces.enabled", true);
pref("layout.css.color-mix.enabled", true);
pref("layout.css.exp.enabled", true);
pref("layout.css.font-display.enabled", true);
pref("layout.css.mod-rem.enabled", true);
pref("layout.css.nan-inf.enabled", true);
pref("layout.css.round.enabled", true);
pref("layout.css.trig.enabled", true);
pref("layout.spammy_warnings.enabled", false);
pref("media.ffvpx.mp3.enabled", true);
pref("media.ffvpx.opus.enabled", true);
pref("media.ffvpx.vorbis.enabled", true);
pref("media.ffvpx.wav.enabled", true);
pref("media.webvtt.pseudo.enabled", true);
pref("security.csp.enableNavigateTo", false);
pref("toolkit.lazyHiddenWindow", true);
```

==CHANGED
```js
pref("browser.firefox-view.search.enabled", true); // prev: false
pref("browser.firefox-view.virtual-list.enabled", true); // prev: false
pref("browser.vpn_promo.disallowed_regions", "ae,by,cn,cu,iq,ir,kp,om,ru,sd,sy,tm,tr"); // prev: "ae,by,cn,cu,iq,ir,kp,om,ru,sd,sy,tm,tr,ua"
pref("dom.webcomponents.shadowdom.declarative.enabled", true); // prev: false
pref("editor.block_inline_check.use_computed_style", true); // prev: false
pref("gfx.canvas.accelerated.max-size", 8192); // prev: 0
pref("identity.fxaccounts.toolbar.defaultVisible", true); // prev: false
pref("javascript.options.spectre.jit_to_cxx_calls", false); // prev: true
pref("media.eme.playready.istypesupportedex", true); // prev: false
pref("media.webvtt.testing.events", false); // prev: true
pref("network.early-hints.enabled", true); // prev: false
pref("network.url.useDefaultURI", false); // prev: true
pref("ui.new-webcompat-reporter.enabled", true); // prev: false
pref("ui.new-webcompat-reporter.reason-dropdown", 2); // prev: 0
```

</p></details>

**Comment by earthlng** [comment]
<details><summary>some bugzilla tickets</summary><p>

* apz.minimap.enabled
Bug [1604280](https://bugzilla.mozilla.org/show_bug.cgi?id=1604280) - Implement an APZ minimap for WebRender.

* apz.scrollend-event.content.enabled
Bug [1872772](https://bugzilla.mozilla.org/show_bug.cgi?id=1872772) - Remove apz.scrollend-event.content.enabled preference.

* browser.firefox-view.search.enabled
Bug [1868853](https://bugzilla.mozilla.org/show_bug.cgi?id=1868853) - Flip search and virtualizer prefs to true
Bug [1826328](https://bugzilla.mozilla.org/show_bug.cgi?id=1826328) - Add search capability to view in history

* browser.firefox-view.virtual-list.enabled
Bug [1868853](https://bugzilla.mozilla.org/show_bug.cgi?id=1868853) - Flip search and virtualizer prefs to true
Bug [1859788](https://bugzilla.mozilla.org/show_bug.cgi?id=1859788) - Implement virtualizer to improve performance of history in Firefox View

* browser.link.force_default_user_context_id_for_external_opens
Bug [1874599](https://bugzilla.mozilla.org/show_bug.cgi?id=1874599) - Add a pref to disable userContextId guessing for external opens;r=mossop  a=pascalc

* browser.ping-centre.log
Bug [1868988](https://bugzilla.mozilla.org/show_bug.cgi?id=1868988) - Remove PingCentre. Its watch has ended.

* browser.ping-centre.telemetry
Bug [1868988](https://bugzilla.mozilla.org/show_bug.cgi?id=1868988) - Remove PingCentre. Its watch has ended.

* browser.search.searchEngineRemoval
Bug [1845150](https://bugzilla.mozilla.org/show_bug.cgi?id=1845150) - Update notificationbox usage for search engine removal notification

* browser.shopping.experience2023.autoOpen.enabled
Bug [1876608](https://bugzilla.mozilla.org/show_bug.cgi?id=1876608) - register autoOpen feature flag in Nimbus and set pref to false by default. a=pascalc
Bug [1876568](https://bugzilla.mozilla.org/show_bug.cgi?id=1876568) - add new pref for auto-open toggle and distinguish from feature flag.

* browser.shopping.experience2023.autoOpen.userEnabled
Bug [1876568](https://bugzilla.mozilla.org/show_bug.cgi?id=1876568) - add new pref for auto-open toggle and distinguish from feature flag.

* browser.shopping.experience2023.showKeepSidebarClosedMessage
Bug [1876200](https://bugzilla.mozilla.org/show_bug.cgi?id=1876200) - Add keep closed shopping message bar.

* browser.shopping.experience2023.sidebarClosedCount
Bug [1876200](https://bugzilla.mozilla.org/show_bug.cgi?id=1876200) - Add keep closed shopping message bar.

* browser.tabs.cardPreview.delayMs
Bug [1783521](https://bugzilla.mozilla.org/show_bug.cgi?id=1783521) - add card preview when inactive tab hovered.

* browser.tabs.cardPreview.enabled
Bug [1783521](https://bugzilla.mozilla.org/show_bug.cgi?id=1783521) - add card preview when inactive tab hovered.

* browser.tabs.cardPreview.showThumbnails
Bug [1783521](https://bugzilla.mozilla.org/show_bug.cgi?id=1783521) - add card preview when inactive tab hovered.

* browser.urlbar.suggest.yelp
Bug [1855375](https://bugzilla.mozilla.org/show_bug.cgi?id=1855375): Basic implementation for Yelp Suggestions

* browser.vpn_promo.disallowed_regions
Bug [1864079](https://bugzilla.mozilla.org/show_bug.cgi?id=1864079) - Update browser.vpn_promo.disallowed_regions pref and regions list in BrowserUtils regarding Ukraine(ua)

* cookiebanners.ui.desktop.showCallout
Bug [1859632](https://bugzilla.mozilla.org/show_bug.cgi?id=1859632) - add feature callout for cookie banner reduction.

* devtools.inspector.rule-view.focusNextOnEnter
Bug [1877457](https://bugzilla.mozilla.org/show_bug.cgi?id=1877457) - [devtools] Default devtools.inspector.rule-view.focusNextOnEnter to true. .  a=pascalc
Bug [1876694](https://bugzilla.mozilla.org/show_bug.cgi?id=1876694) - [devtools] Add preference to make Rules view enter key move focus to next input. .  a=pascalc
Bug [1877457](https://bugzilla.mozilla.org/show_bug.cgi?id=1877457) - [devtools] Default devtools.inspector.rule-view.focusNextOnEnter to true. .  a=dmeehan
Bug [1876694](https://bugzilla.mozilla.org/show_bug.cgi?id=1876694) - [devtools] Add preference to make Rules view enter key move focus to next input. .  a=dmeehan

* devtools.inspector.showRulesViewEnterKeyNotice
Bug [1877457](https://bugzilla.mozilla.org/show_bug.cgi?id=1877457) - [devtools] Remove ruleview-kbd-enter-notice. .  a=pascalc
Bug [1871806](https://bugzilla.mozilla.org/show_bug.cgi?id=1871806) - [devtools] Add notice for Enter behavior in Rules view.
Bug [1877457](https://bugzilla.mozilla.org/show_bug.cgi?id=1877457) - [devtools] Remove ruleview-kbd-enter-notice. .  a=dmeehan
Bug [1871806](https://bugzilla.mozilla.org/show_bug.cgi?id=1871806) - [devtools] Add notice for Enter behavior in Rules view.  a=dmeehan.

* dom.importMaps.enabled
Bug [1853547](https://bugzilla.mozilla.org/show_bug.cgi?id=1853547) - Remove dom.importMaps.enabled pref

* dom.mouse_event.region.enabled
Bug [1853517](https://bugzilla.mozilla.org/show_bug.cgi?id=1853517) - Remove dom.mouse_event.region.enabled pref

* dom.webcomponents.shadowdom.declarative.enabled
Bug [1870052](https://bugzilla.mozilla.org/show_bug.cgi?id=1870052) - Enable Declarative ShadowDOM on all channels.
Bug [1712140](https://bugzilla.mozilla.org/show_bug.cgi?id=1712140) - Part 5: Block Declarative ShadowDOM behind feature pref.

* dom.webidl.crosscontext_hasinstance.enabled
Bug [1870458](https://bugzilla.mozilla.org/show_bug.cgi?id=1870458) - Remove always-disabled dom.webidl.crosscontext_hasinstance.enabled pref.

* editor.block_inline_check.use_computed_style
Bug [1858071](https://bugzilla.mozilla.org/show_bug.cgi?id=1858071) - Make `HTMLEditor` consider whether block or inline with CSS in the release channel

* editor.inline_style.range.compatible_with_the_other_browsers
Bug [1874806](https://bugzilla.mozilla.org/show_bug.cgi?id=1874806) - Get rid of `editor.inline_style.range.compatible_with_the_other_browsers` pref

* editor.join_split_direction.force_use_compatible_direction
Bug [1873307](https://bugzilla.mozilla.org/show_bug.cgi?id=1873307) - Remove unused editor.join_split_direction.force_use_traditional_direction and editor.join_split_direction.force_use_compatible_direction prefs

* editor.join_split_direction.force_use_traditional_direction
Bug [1873307](https://bugzilla.mozilla.org/show_bug.cgi?id=1873307) - Remove unused editor.join_split_direction.force_use_traditional_direction and editor.join_split_direction.force_use_compatible_direction prefs

* extensions.formautofill.addresses.capture.requiredFields
Bug [1873215](https://bugzilla.mozilla.org/show_bug.cgi?id=1873215) - Display address capture prompt only for forms with all the required fields

* extensions.formautofill.heuristics.captureOnPageNavigation
Bug [1870718](https://bugzilla.mozilla.org/show_bug.cgi?id=1870718) - P1. Add heuristic to capture address fields on page navigation

* gfx.canvas.accelerated.max-size
Bug [1829026](https://bugzilla.mozilla.org/show_bug.cgi?id=1829026) - Support using DrawTargetWebgl via remote canvas.

* gfx.canvas.remote.texture-timeout-ms
Bug [1871467](https://bugzilla.mozilla.org/show_bug.cgi?id=1871467) - Remove unused ReplayTexture support.

* identity.fxaccounts.oauth.enabled
Bug [1869641](https://bugzilla.mozilla.org/show_bug.cgi?id=1869641): Adds oauth handling for fxa behind pref.

* identity.fxaccounts.toolbar.defaultVisible
Bug [1874311](https://bugzilla.mozilla.org/show_bug.cgi?id=1874311): Enables Account toolbar by default when signed out.

* javascript.options.destructuring_fuse
Bug [1870396](https://bugzilla.mozilla.org/show_bug.cgi?id=1870396) - Add pref for fuse-guarded destructuring

* javascript.options.spectre.jit_to_cxx_calls
Bug [1877605](https://bugzilla.mozilla.org/show_bug.cgi?id=1877605). Disable spectreJitToCxxCalls everywhere.

* javascript.options.watchtower.megamorphic
Bug [1872656](https://bugzilla.mozilla.org/show_bug.cgi?id=1872656) - Remove megamorphic watchtower option

* layout.css.abs-sign.enabled
Bug [1874633](https://bugzilla.mozilla.org/show_bug.cgi?id=1874633) - Remove a bunch of css math prefs

* layout.css.color-mix.color-spaces.enabled
Bug [1873594](https://bugzilla.mozilla.org/show_bug.cgi?id=1873594) - Remove layout.css.color-mix.enabled and layout.css.color-mix.color-spaces.enabled

* layout.css.color-mix.enabled
Bug [1873594](https://bugzilla.mozilla.org/show_bug.cgi?id=1873594) - Remove layout.css.color-mix.enabled and layout.css.color-mix.color-spaces.enabled

* layout.css.exp.enabled
Bug [1874633](https://bugzilla.mozilla.org/show_bug.cgi?id=1874633) - Remove a bunch of css math prefs

* layout.css.font-display.enabled
Bug [1872784](https://bugzilla.mozilla.org/show_bug.cgi?id=1872784) - Remove layout.css.font-display.enabled pref

* layout.css.grid-subgrid-baselines.enabled
Bug [1833482](https://bugzilla.mozilla.org/show_bug.cgi?id=1833482) part 1: Propagate baseline offset metrics from temporary GridItemInfo structs in parent grid to the canonical GridItemInfo structs in the subgrids.

* layout.css.mod-rem.enabled
Bug [1874633](https://bugzilla.mozilla.org/show_bug.cgi?id=1874633) - Remove a bunch of css math prefs

* layout.css.nan-inf.enabled
Bug [1874633](https://bugzilla.mozilla.org/show_bug.cgi?id=1874633) - Remove a bunch of css math prefs

* layout.css.round.enabled
Bug [1874633](https://bugzilla.mozilla.org/show_bug.cgi?id=1874633) - Remove a bunch of css math prefs

* layout.css.trig.enabled
Bug [1874633](https://bugzilla.mozilla.org/show_bug.cgi?id=1874633) - Remove a bunch of css math prefs

* layout.cursor.disable-for-popups
Bug [1833814](https://bugzilla.mozilla.org/show_bug.cgi?id=1833814) - change nsIWidget's cursor logic,  a=pascalc

* layout.spammy_warnings.enabled
Bug [1873311](https://bugzilla.mozilla.org/show_bug.cgi?id=1873311) - Remove unused layout.spammy_warnings.enabled pref

* media.eme.playready.istypesupportedex
Bug [1825142](https://bugzilla.mozilla.org/show_bug.cgi?id=1825142) - part6 : enable the pref 'media.eme.playready.istypesupportedex'.
Bug [1851914](https://bugzilla.mozilla.org/show_bug.cgi?id=1851914) - part1 : guard IsTypeSupportedEx under a pref.

* media.eme.wmf.clearkey.enabled
Bug [1825142](https://bugzilla.mozilla.org/show_bug.cgi?id=1825142) - part1 : use Media Foundation Clearkey CDM when the pref is on.

* media.eme.wmf.use-mock-cdm-for-external-cdms
Bug [1825142](https://bugzilla.mozilla.org/show_bug.cgi?id=1825142) - part5 : use mock CDM (clearkey) to test all supported key systems.

* media.ffvpx.mp3.enabled
Bug [1873663](https://bugzilla.mozilla.org/show_bug.cgi?id=1873663) - Remove legacy audio decoders (vorbis, opus, pcm).

* media.ffvpx.opus.enabled
Bug [1873663](https://bugzilla.mozilla.org/show_bug.cgi?id=1873663) - Remove legacy audio decoders (vorbis, opus, pcm).

* media.ffvpx.vorbis.enabled
Bug [1873663](https://bugzilla.mozilla.org/show_bug.cgi?id=1873663) - Remove legacy audio decoders (vorbis, opus, pcm).

* media.ffvpx.wav.enabled
Bug [1873663](https://bugzilla.mozilla.org/show_bug.cgi?id=1873663) - Remove legacy audio decoders (vorbis, opus, pcm).

* media.webvtt.pseudo.enabled
Bug [1874618](https://bugzilla.mozilla.org/show_bug.cgi?id=1874618) - Remove media.webvtt.pseudo.enabled pref

* media.webvtt.testing.events
Bug [1874738](https://bugzilla.mozilla.org/show_bug.cgi?id=1874738) - Unship non-standard mozStartedLoadingTextTrack event

* network.cookie.prevent_set_cookie_from_multipart
Bug [1864385](https://bugzilla.mozilla.org/show_bug.cgi?id=1864385) - Block set-cookie from multipart/x-mixed-replace with pref.

* network.dns.native_https_query
Bug [1873461](https://bugzilla.mozilla.org/show_bug.cgi?id=1873461) - Add pref for resolving HTTPS records using native resolver on win 10
Bug [1852900](https://bugzilla.mozilla.org/show_bug.cgi?id=1852900) - Use DNSQuery_A to resolve HTTPS record on Windows

* network.dns.native_https_query_win10
Bug [1873461](https://bugzilla.mozilla.org/show_bug.cgi?id=1873461) - Add pref for resolving HTTPS records using native resolver on win 10

* network.dns.native_https_timeout_android
Bug [1852900](https://bugzilla.mozilla.org/show_bug.cgi?id=1852900) - Use android_res_nquery from libandroid.so to resolve HTTPS records on Android (API level &gt;= 29)

* network.early-hints.enabled
Bug [1874445](https://bugzilla.mozilla.org/show_bug.cgi?id=1874445) - Enable Early Hints preload

* network.fetch.cache_partition_cross_origin
Bug [1816390](https://bugzilla.mozilla.org/show_bug.cgi?id=1816390) - Partition the cache entries generated by cross-origin fetch requests

* network.http.useragent.freezeCpu
Bug [1861847](https://bugzilla.mozilla.org/show_bug.cgi?id=1861847) - Hard code User-Agent string's CPU arch on Android, Linux, and other Unix-like platforms.

* network.protocol-handler.external.firefox
Bug [1873223](https://bugzilla.mozilla.org/show_bug.cgi?id=1873223) - Modified firefox and firefox-private protocol handler behavior

* network.protocol-handler.external.firefox-private
Bug [1873223](https://bugzilla.mozilla.org/show_bug.cgi?id=1873223) - Modified firefox and firefox-private protocol handler behavior

* network.standard_content_type_parsing.response_headers
Bug [1847659](https://bugzilla.mozilla.org/show_bug.cgi?id=1847659) - Use the new MimeType parser to parse content-types for response headers;

* network.url.useDefaultURI
Bug [1877753](https://bugzilla.mozilla.org/show_bug.cgi?id=1877753) - Disable DefaultURI for unknown schemes.
Bug [1603699](https://bugzilla.mozilla.org/show_bug.cgi?id=1603699) - Enable default URI pref.

* privacy.clearOnShutdown_v2.cache
Bug [1861450](https://bugzilla.mozilla.org/show_bug.cgi?id=1861450) - (Part 1) Create new prefs and categories in Sanitizer for the new dialog box.

* privacy.clearOnShutdown_v2.cookiesAndStorage
Bug [1861450](https://bugzilla.mozilla.org/show_bug.cgi?id=1861450) - (Part 1) Create new prefs and categories in Sanitizer for the new dialog box.

* privacy.clearOnShutdown_v2.historyFormDataAndDownloads
Bug [1869884](https://bugzilla.mozilla.org/show_bug.cgi?id=1869884) - Remerging downloads into history.

* privacy.clearOnShutdown_v2.siteSettings
Bug [1861450](https://bugzilla.mozilla.org/show_bug.cgi?id=1861450) - (Part 1) Create new prefs and categories in Sanitizer for the new dialog box.

* privacy.partition.network_state.ocsp_cache
Bug [1874143](https://bugzilla.mozilla.org/show_bug.cgi?id=1874143) - Let privacy.partition.network_state.ocsp_cache ride the trains

* privacy.sanitize.useOldClearHistoryDialog
Bug [1856415](https://bugzilla.mozilla.org/show_bug.cgi?id=1856415) - (Part 1) Updating clear data dialog UI.

* screenshots.browser.component.last-saved-method
Bug [1867644](https://bugzilla.mozilla.org/show_bug.cgi?id=1867644) - Focus last used button for screenshots UI.

* security.csp.enableNavigateTo
Bug [1793560](https://bugzilla.mozilla.org/show_bug.cgi?id=1793560) - Remove navigate-to CSP directive

* security.tls.enable_kyber
Bug [1874962](https://bugzilla.mozilla.org/show_bug.cgi?id=1874962) - add Xyber768 support under a pref.

* services.sync.prefs.sync.privacy.clearOnShutdown_v2.cache
Bug [1861450](https://bugzilla.mozilla.org/show_bug.cgi?id=1861450) - (Part 1) Create new prefs and categories in Sanitizer for the new dialog box.

* services.sync.prefs.sync.privacy.clearOnShutdown_v2.cookiesAndStorage
Bug [1861450](https://bugzilla.mozilla.org/show_bug.cgi?id=1861450) - (Part 1) Create new prefs and categories in Sanitizer for the new dialog box.

* services.sync.prefs.sync.privacy.clearOnShutdown_v2.downloads
Bug [1861450](https://bugzilla.mozilla.org/show_bug.cgi?id=1861450) - (Part 1) Create new prefs and categories in Sanitizer for the new dialog box.

* services.sync.prefs.sync.privacy.clearOnShutdown_v2.historyFormDataAndDownloads
Bug [1869884](https://bugzilla.mozilla.org/show_bug.cgi?id=1869884) - Remerging downloads into history.

* services.sync.prefs.sync.privacy.clearOnShutdown_v2.siteSettings
Bug [1861450](https://bugzilla.mozilla.org/show_bug.cgi?id=1861450) - (Part 1) Create new prefs and categories in Sanitizer for the new dialog box.

* toolkit.lazyHiddenWindow
Bug [1872824](https://bugzilla.mozilla.org/show_bug.cgi?id=1872824) - Remove toolkit.lazyHiddenWindow pref
Bug [1857049](https://bugzilla.mozilla.org/show_bug.cgi?id=1857049) - Move toolkit.lazyHiddenWindow pref to all.js

* ui.new-webcompat-reporter.enabled
Bug [1879130](https://bugzilla.mozilla.org/show_bug.cgi?id=1879130) - enable Report Broken Site by default, with 'reason' drop-down required;
Bug [1852340](https://bugzilla.mozilla.org/show_bug.cgi?id=1852340) - part 1: Add a new Gecko component for reporting broken websites;

* ui.new-webcompat-reporter.reason-dropdown
Bug [1879130](https://bugzilla.mozilla.org/show_bug.cgi?id=1879130) - enable Report Broken Site by default, with 'reason' drop-down required;
Bug [1876880](https://bugzilla.mozilla.org/show_bug.cgi?id=1876880) - put the randomization of Report Broken Site reason-dropdown behind a pref, and base its ordering on ClientEnvironment.randomizationId;
Bug [1852340](https://bugzilla.mozilla.org/show_bug.cgi?id=1852340) - part 1: Add a new Gecko component for reporting broken websites;

* ui.new-webcompat-reporter.reason-dropdown.randomized
Bug [1876880](https://bugzilla.mozilla.org/show_bug.cgi?id=1876880) - put the randomization of Report Broken Site reason-dropdown behind a pref, and base its ordering on ClientEnvironment.randomizationId;

</p></details>

**Comment by rusty-snake** [comment]
> ```js
> pref("browser.link.force_default_user_context_id_for_external_opens", false);
> ```

:tada:

**Comment by Thorin-Oakenpants** [comment]
> These prefs only affect/hide the button

yup, totally get that :)

> I don't consider View to be a privacy/security issue

we're not TB and they don't want to enable any outgoing connections if they can help it because mumble mumble timing attacks circuits mumble etc. But that's not why they did this: it requires a Mozilla account which base browser users don't have, so they cleaned up the UI - a button and an about page :)  even though "Firefox View does not appear in PB windows"

https://support.mozilla.org/en-US/kb/how-set-tab-pickup-firefox-view
- unless I'm missing something, I don't see any privacy issues. If you chose to sync, it's all E2EE (unless I'm missing something)

**Comment by Thorin-Oakenpants** [comment]
nice: https://gitlab.torproject.org/tpo/applications/tor-browser/-/issues/41309#note_3006995
- `screenshots.browser.component.enabled` = true in nightly - seems to work flawlessly so far
- meta https://bugzilla.mozilla.org/show_bug.cgi?id=1696573

**Comment by Thorin-Oakenpants** [comment]
loving the silence and stability :)

![silenceisgolden](https://github.com/arkenfox/user.js/assets/16656956/bc984649-8bb3-4557-86c5-8fa6fae8f3d4)

**Comment by crssi** [comment]
@Thorin-Oakenpants you (and the gang) tweaked it to the maximum perfection. Thank you all ❤️

**Comment by Thorin-Oakenpants** [comment]
new
- pref("javascript.options.wasm_moz_intgemm", false);

https://bugzilla.mozilla.org/show_bug.cgi?id=1722102#c19 😕

**Comment by Thorin-Oakenpants** [comment]
@rusty-snake I don't use containers .. can you please craft this for me and tell me if it should be active/commented out and what the value should be - TIA 🧀 

```
/* 1703: */
user_pref("browser.link.force_default_user_context_id_for_external_opens", true);

```

**Comment by rusty-snake** [comment]
Value: `false` is the default / "new" behaviour, `true` restores the "old" behaviour before firefox 120(?). You want `true` if you use TC (automatic), MAC "Always open in ..." or want only selected "sessions" in a container while using the default container by default. You want `false` if you want all "sessions" in a/the container. The bugzilla explains it.
active/commented out IDK.

```js
/* 1703: Always open (external links|links from external applications) in the (default container|without a container) [FF123+]
 * Some container extensions need this to work correctly. ***/
user_pref("browser.link.force_default_user_context_id_for_external_opens", true);
```

**Comment by Thorin-Oakenpants** [comment]
thanks, I guess. I can't really parse that

**Comment by Thorin-Oakenpants** [comment]
OK, I'll try again. It's going to be inactive, I am adding it for the info factor. Containers are a cool tool for (extra) partitioning - hence adding it, and hence we enable it. **_But_** I don't use containers, MAC, TC or any of that - and I have no idea, or desire, to try and understand the workflows or intricacies of WTF is going on

someone will need to write something that ELI5 for readers - for example, I literally cannot parse the previous attempt, and I'm not going to go read some long bugzilla (as a user)

Until I can add this, it's a blocker for any more updates - thanks

**Comment by Thorin-Oakenpants** [comment]
PS .. loving my TB 126 😁 

![tada](https://github.com/arkenfox/user.js/assets/16656956/b82df381-ef75-4940-a52e-152cd8c29e81)

**Comment by Thorin-Oakenpants** [comment]
```js
/* 1703: Always open (external links|links from external applications) in the (default container|without a container) [FF123+]
 * Some container extensions need this to work correctly. ***/
user_pref("browser.link.force_default_user_context_id_for_external_opens", true);
```

^ still not easily readable or understandable. I cannot tell WTF true or false applies to. External links and links from external applications is the same thing. So `true` means open in a default container AND no container? What does false do?. Why are we using pipes? What does "work correctly" mean?

```js
/* 1703: set how containers work with external links [FF123]
 * [SETUP-WEB] Depending on your container extension(s) and their settings
 * true=Firefox will not choose a container (so your extension can)
 * false=Firefox will choose the container (default)
   // user_pref("browser.link.force_default_user_context_id_for_external_opens", true);
```

Come on you guys ... I can't write everything. This doesn't even seem correct
- `true` - are _you_ (readers) sure that = NO container, ever?
   - what if containers aren't enabled (doh)
   - see rusty's example above which seems to suggest it can do no container or the default container
- `false` - are _you_ sure that = FF will always choose a container? Which one?
   - what if containers aren't enabled

I literally cannot write this shit or deep dive it - and I refuse to. Either help or I won't bother adding it - thanks

**Comment by Thorin-Oakenpants** [comment]
updated [image](https://github.com/arkenfox/user.js/issues/1814#issuecomment-1993997501) of silence

**Comment by Thorin-Oakenpants** [comment]
> alternatives to choose from

they're the complete opposite of each other - `default container|without a container` - hence it's unparsable

appreciate the help .. will come back to this after I get back from portugal

**Comment by Thorin-Oakenpants** [comment]
Oh come on ... it's ambiguous at best. The word container implies a .. wait for it .. container .. and without a container implies .. wait for it .. no container = they are not the same. `context` - what does that mean (to readers)? It's not going to help just changing a word

I can't add ambiguous info - it needs to be crystal clear to the reader what is going to happen

**Comment by Thorin-Oakenpants** [comment]
> The one which most of the pages of the same domain are currently opened in

this is better, we could add some `e.g.` lines to the entry

**Comment by rusty-snake** [comment]
"without a container": What the user sees and think it happens.
"default container": What technically happens.

work correctly: (examples)
- TC (automatic): Use a new temporary container rather than reusing some existing one.
- MAC: Follow "always open in" even if the user manually opened this site in an other container.

> > Which one?
> The one which most of the pages of the same domain are currently opened in:

This heuristic will be expanded in the future. When containers are part of history, this will be taken into account too for example.

**Comment by Thorin-Oakenpants** [comment]
> default container|without a container

in other words, the default container is no container - this is redundant

**Comment by Thorin-Oakenpants** [comment]
https://github.com/arkenfox/user.js/pull/1816/commits/13f063dc95963fc21cc6b3fc9c5f13f2ff9cac5e

speak now if you have issues with it - I will probably land 126 today or tomorrow
